config/software/expat.rb

```diff
@@ -24,8 +24,12 @@
 license_file "COPYING"
 skip_transitive_dependency_licensing true
 
-source url: "http://downloads.sourceforge.net/project/expat/expat/#{version}/expat-#{version}.tar.gz",
-       md5: "dd7dab7a5fea97d2a6a43f511449b7cd"
+# version_list: url=https://sourceforge.net/projects/expat/files/expat/#{version}/ filter=*.tar.gz
```

Comment:
I'm not sure that URL would work to find new versions, as it is incorporating the version into the URL itself. I think the best URL to find new versions would be their GitHub releases page: https://github.com/libexpat/libexpat/releases

Reply:
Updated the url for version list